src/Microsoft.FeatureManagement/FilterAliasAttribute.cs

```diff
@@ -6,7 +6,7 @@
 namespace Microsoft.FeatureManagement
 {
     /// <summary>
-    /// Allows the name of an <see cref="IFeatureFilter"/> to be customized to relate to the name specified in configuration.
+    /// Allows the name of an <see cref="IFeatureFilter"/> or <see cref="IFeatureVariantAssigner"/> to be customized to relate to the name specified in configuration.
```

Comment:
I know it will be nice to reuse, but it's kind of odd to use "FilterAlias" for "VariantAssigner". Probably, we can come up a new attribute that can be used for both and keep this one just for legacy compatibility?

Reply:
How about just `Alias`?

Reply:
Ideally, we don't introduce a name with an existing type [https://docs.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.web.codegeneration.commandline.aliasattribute?view=aspnetcore-2.2&viewFallbackFrom=aspnetcore-5.0](https://docs.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.web.codegeneration.commandline.aliasattribute?view=aspnetcore-2.2&viewFallbackFrom=aspnetcore-5.0)

Alternatively, we introduce a `AssignerAliasAttribute`. We have to keep `FilterAliasAttribute` anyway, so it's not too bad. We can have both attributes to derive from a common type to share code.

Reply:
> Alternatively, we introduce a AssignerAliasAttribute.

I'm fine with that.

> We can have both attributes to derive from a common type to share code.

Not sure what that type would be.

> Ideally, we don't introduce a name with an existing type

It doesn't seem like that big of an issue. If it was a system type, then I'd be a bit opposed, but that's visual studio code generation, an isolated namespace. Even in the event that a customer was using that namespace in a featurefilter/assigner the only effect would be namespace qualification.

Reply:
> Not sure what that type would be.

I was just talking about code reuse if you don't want to write the same code twice. For example, you can create a private `AliasAttribute` and have both filter and assigner attributes derived from it. The processing side of code hopefully can just look for the `AliasAttribute` too.

Reply:
Can a public type derive from a private type?

Reply:
Alright, I guess we can have the assigner attribute derived from the filter attribute for code reuse.

Reply:
I'll add AssignerAliasAttribute, but not sure about having it extend FilterAlias attribute. Doesn't seem like proper inheritance.

Reply:
Added.